aunt-leahs-app/src/App.js

```diff
@@ -1,15 +1,16 @@
 import React from 'react';
 
 import './App.css';
-import AdminLogin from './pages/adminLogin';
-
+import HomePage from './pages/homepage/homepage';
+import VolunteerData from './pages/VolunteerData';
+import ShiftDataPage from './pages/ShiftDataPage';
 
 function App() {
-  return (
-    <div className='testtest'>
-        <AdminLogin />
-    </div>
-  );
+	return (
```

Comment:
Don't modify App.js for testing purposes. You should remove your test code after you're done.